terraform/modules/kafka/main.tf

```diff
@@ -0,0 +1,42 @@
+resource "aws_cloudwatch_log_group" "msk_broker_logs" {
+  name = "msk_broker_logs"
+}
+
+resource "aws_msk_cluster" "kafka" {
+  cluster_name           = "kafka-eshop"
```

Comment:
should be a variable just like we did in the other modules

Reply:
This will not be a public reusable module. We could define `cluster_name` though it would be set to one value always. WDTY ?